src/scss/_mixins.scss

```diff
@@ -243,16 +239,20 @@
 		background-color: $background;
 	}
 
-	@if not $is-light {
+	@if not $link-contrasts-with-background {
+		$hover-color: darken($foreground, 20%);
+
 		.o-topper__standfirst,
 		.o-topper__summary {
 			a {
 				color: $foreground;
 				border-bottom-color: $foreground;
+				text-decoration-color: $foreground;
 
 				&:hover {
 					color: $hover-color;
 					border-bottom-color: $hover-color;
+					text-decoration-color: $hover-color;
 				}
```

Comment:
I forget to mention `oTypographyLink` accepts themes, there's some logic in there to create a slightly off colour underline, set focus styles, etc
```suggestion
				@include oTypographyLink($theme: (
				 	'base': $foreground,
					'hover': $hover-color,
					'context': $background,
				), $include-base-styles: false);
```

Reply:
Actually, the contrast on the underline using `oTypographyLink($theme)` is really low 😞  let's stick with what you have 👍 and I'll open a o-typography issue
![Screenshot 2021-02-26 at 17 52 55](https://user-images.githubusercontent.com/10405691/109336565-7d088700-785b-11eb-8589-5c91cb2c0057.png)